tests/integration/kubernetes/k8s-exec-rejected.bats

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2023 Microsoft.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../common.bash"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+
+setup() {
+	get_pod_config_dir
+	pod_name="busybox"
+	pod_yaml="${pod_config_dir}/busybox-pod.yaml"
+
+	# String generated using "base64 -w 0 kata-containers/src/kata-opa/allow-all-except-exec-process.rego"
```

Comment:
Wouldn't it be better to actually generate this string from the file? Or is the file or `base64` not available at runtime?

Reply:
I agree but I wasn't familiar enough with the test Host environment - didn't know how easy it is to find base64 on these Hosts. There is always room left for improvements in these tests.